### ISSUE TITLE
Возвращение имен на русском

### ISSUE THREAD

**Issue by Veilvi** [VKCOM/vk-android-sdk#464]
SDK 2.2.3 Возвращает имя пользователя на английском. 

class VKWallRequest : VKRequest<List<VKWall>> {

    constructor(uid:Int=0) : super("wall.get") {
        val config= VKApiConfig
        addParam("owner_id",uid)
        addParam("extended",1)
        addParam("count",100)
        addParam("lang","ru")
    }

    override fun parse(r: JSONObject): ArrayList<VKWall> {}
Параметр lang затирается, пытаюсь создать кастомный конфиг, но не могу получить доступ к lang, а только к полям Companion, так как нет дефолтных значений для всех полей dataclass.

**Comment by Veilvi** [comment]
Переопределил метод onExecute

       override fun onExecute(manager: VKApiManager): List<VKWall> {
        val config = manager.config

        params["lang"] = "ru"
        params["device_id"] = config.deviceId.value
        params["v"] = config.version

        val callBuilder = createBaseCallBuilder(config)
            .args(params)
            .method(method)
            .version(config.version)
        return manager.execute(callBuilder.build(), this)
    }

**Comment by UrbanSide** [comment]
- [X] https://github.com/VKCOM/vk-android-sdk/issues/463

**Comment by UrbanSide** [comment]
@Veilvi ,день добрый,подскажите вы в вашем методе получаете посты со стены?Если не секрет,не могли бы рассказать как?А то второй день мануал курю,не могу с группы своей посты получить.

**Comment by Veilvi** [comment]
Да, со стены. Метод wall.get(). Попробуйте покурить их исходники здесь. Там есть exemple. По примеру user вроде. Там понятнее будет, в документации они пишут про старые версии sdk скорее всего

**Comment by UrbanSide** [comment]
> Да, со стены. Метод wall.get(). Попробуйте покурить их исходники здесь. Там есть exemple. По примеру user вроде. Там понятнее будет, в документации они пишут про старые версии sdk скорее всего

Да вот я как раз по примеру пытался,но что-то все время не так xD

**Comment by Veilvi** [comment]
Это надо обсуждать)). Я тут тоже ковыряюсь.

**Comment by Veilvi** [comment]
> > Да, со стены. Метод wall.get(). Попробуйте покурить их исходники здесь. Там есть exemple. По примеру user вроде. Там понятнее будет, в документации они пишут про старые версии sdk скорее всего
> 
> Да вот я как раз по примеру пытался,но что-то все время не так xD

Дискорд бы, ибо код надо смотреть

**Comment by JaneySprings** [comment]
Для того, чтобы каждый раз это не указывать в запросе, до метода VK.Login можно вызвать VK.setConfig для установки пользовательской конфигурации SDK. Код будет выглядеть так:
`VK.setConfig(VKApiConfig(
            context = this,
            validationHandler = VKDefaultValidationHandler(this),
            version = "5.126",
            lang = "ru" //en
        ))`
Для языка удобнее использовать getString(R.string.ИМЯ_РЕСУРСА), так как при разных системных языках будет изменятся strings.xml файл, откуда и будет браться строковый ресурс.
